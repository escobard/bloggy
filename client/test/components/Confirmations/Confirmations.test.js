import React, { Component } from 'react'
import JobReview from '../../../src/components/Confirmations';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'

describe('JobReview snapshot', () => {
	const initialState = {output:100}
	const mockStore = configureStore()
	let store,container

    beforeEach(()=>{
        store = mockStore(initialState)
        container = shallow(<JobReview /> )  
    })

    it('+++ render the JobReview component', () => {
       expect(toJson(container)).toMatchSnapshot()
    });

});