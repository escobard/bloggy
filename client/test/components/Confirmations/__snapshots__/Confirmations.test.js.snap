// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobReview snapshot +++ render the JobReview component 1`] = `
<div>
  <h2>
    Review Job
  </h2>
</div>
`;
